src/utils.ts

```diff
@@ -34,6 +34,15 @@ export function getRequiredRosDistributions(): string[] {
 		requiredRosDistributionsList = requiredRosDistributions.split(
 			RegExp("\\s")
 		);
+	
+		for(var i = 0; i < requiredRosDistributionsList.length; i++){
+			if(requiredRosDistributionsList[i] != "foxy" && requiredRosDistributionsList[i] != "kinetic" && requiredRosDistributionsList[i] != "melodic" && requiredRosDistributionsList[i] != "dashing" && requiredRosDistributionsList[i] != "eloquent"){
```

Comment:
instead of coding the valid values (data) into this check (functionality), do we already have defined a list of valid distributions somewhere? If not let's define it. This should probably look something like:

```
for (requiredDistro in requiredRosDistributionsList) {
  if (requiredDistro not in validRosDistributions) {
    error("Distro " + requiredDistro + "is not a supported ROS distribution. It must be one of " + validRosDistributions);
} }
```
```